reposerver/repository/repository.go

```diff
@@ -1188,6 +1189,10 @@ func helmTemplate(appPath string, repoRoot string, env *v1alpha1.Env, q *apiclie
 		}
 		passCredentials = appHelm.PassCredentials
 		templateOpts.SkipCrds = appHelm.SkipCrds
+
+		if !appHelm.SkipTests {
+			templateOpts.SkipTests = false
+		}
```

Comment:
I don't think there is a way to differentiate if the boolean is omitted or set to false. To be able to do that, you need to use a boolean pointer and check if it is nil first.